template/src/bin/main.rs

```diff
@@ -55,6 +55,8 @@ fn main() -> ! {
     esp_println::logger::init_logger_from_env();
     //ENDIF
 
+    esp_bootloader_esp_idf::esp_app_desc!();
```

Comment:
I think a small comment, saying why we need this (for esp-idf bootloader support) will be helpful to new users